studio/components/interfaces/Reports/ReportWidget.tsx

```diff
@@ -16,6 +16,7 @@ export interface ReportWidgetProps<T = any> {
   renderer: (props: ReportWidgetProps) => React.ReactNode
   params: BaseReportParams | LogsEndpointParams
   isLoading: boolean
+  router: NextRouter
```

Comment:
any reason why we're passing the router as props, but within ReportWidget we're declaring another instance of router = useRouter()?